astropixie-widgets/astropixie_widgets/config.py

```diff
@@ -41,26 +41,12 @@ def setup_notebook(debug=False, local=False):
         logging.basicConfig(level=logging.DEBUG)
         logging.debug('Running notebook in debug mode.')
 
-    if local:
-        global jupyter_proxy_url
+    # If JUPYTERHUB_SERVICE_PREFIX environment variable isn't set,
+    # this means that you're running JupyterHub not with Hub in k8s,
+    # and not using run_local.sh (which sets it to empty).
+    if 'JUPYTERHUB_SERVICE_PREFIX' not in os.environ:
         jupyter_proxy_url = 'localhost:8888'
```

Comment:
Ideally we could use 8080 which is the same port local jupyter lab uses by default.